cupy/linalg/_eigenvalue.py

```diff
@@ -66,10 +66,13 @@ def _syevd(a, UPLO, with_eigen_vector):
 
 
 def eigh(a, UPLO='L'):
-    """Eigenvalues and eigenvectors of a symmetric matrix.
+    """
+    Return the eigenvalues and eigenvectors of a complex Hermitian
+    (conjugate symmetric) or a real symmetric matrix.
```

Comment:
:+1:

The shorter (one-line) summaries were because CuPy had followed OpenStack Hacing.  Now the page of the function list looks nice thanks to `pydata_sphinx_theme`.
https://cupy--5494.org.readthedocs.build/en/5494/reference/linalg.html